### DISCUSSION TITLE
Adding sensors manually to Home Assistant

### DISCUSSION THREAD

**Discussion by dennycol** [aquaticus/nexus433#37] [General]
Hi, congratulations on the great project, I would like to ask you for help if you can,
I have a 433 Mhz receiver like yours and two Digoo DG-R8H, I made the connections on the raspberry and followed the instructions to the letter, the sensors are recognized and work, but I cannot integrate them into Home Assistant, I added the configuration in the ' cinfiguration.yaml
mqtt:
  discovery: true
And
- platform: mqtt
  state_topic: 'nexus433 / sensor / 0xf190aaf29_f101 / state'
  name: 'Bedroom Humidity'
  expire_after: 90
  unit_of_measurement: '%'
  device_class: humidity
  availability_topic: 'nexus433 / sensor / 0xf190aaf29_f101 / connection'
  value_template: '{{value_json.humidity}}'

but to no avail, where am I wrong?
thank you

![1](https://user-images.githubusercontent.com/99897364/157636271-971d9393-b3f1-4ff9-8440-ed0a07c9c4a5.jpg)

**Comment by aquaticus** [comment]
Hi,

First of all, make sure you restarted Home Assistant. When you modify configration file this is required step to apply changes.

The second thing, make sure you see MQTT messages, use something like `mosquitto_sub -v -t "#"`.

I see you got spaces in state and availibility topic: `state_topic: 'nexus433 / sensor / 0xf190aaf29_f101 / state'`, make sure there are no any spaces, it sould be: `state_topic: 'nexus433/sensor/0xf190aaf29_f101/state'`

**Comment by dennycol** [comment]
Thanks for the reply,

yes the MQTT messages are seen, the spaces in the configuarion.yaml are not actually there, the syntax seems correct but I still don't see any sensor in home assistant.
thanks anyway for your availability

regards

**Reply by aquaticus** [reply]
jus a stupid thing. Did you enable MQTT integration?

**Reply by dennycol** [reply]
If the MQTT integration is enabled, I will try to repeat the whole procedure

thank you
